ui/app/src/Views/AlertList/Views.elm

```diff
@@ -98,23 +98,23 @@ defaultAlertGroups activeId activeGroups expandAll groups =
                 labels_ =
                     Dict.toList labels
             in
-            alertGroup activeId (Set.singleton labels_) labels_ alerts expandAll
+            alertGroup activeId (Set.singleton 0) labels_ alerts 0 expandAll
 
         _ ->
             div [ class "pl-5" ]
-                (List.map
-                    (\{ labels, alerts } ->
-                        alertGroup activeId activeGroups (Dict.toList labels) alerts expandAll
-                    )
-                    groups
+                (List.indexedMap Tuple.pair groups
+                    |> List.map
+                        (\( index, group ) ->
```

Comment:
There is no need to first do `indexedMap` to build tuples and then map over tuples. You can call `List.indexedMap` with `\index group ->`.

Reply:
Thanks!